src/actions/speaker-actions.js

```diff
@@ -873,8 +873,22 @@ export const sendSpeakerEmails = (currentFlowEvent,
         };
 
         if (promoCodeStrategy === 4) {
```

Comment:
lets use a constant here 4 seems to be a magic number

Reply:
done